src/components/EditRecipe/EditRecipeForm/Step1.jsx

```diff
@@ -193,7 +193,7 @@ function Step1(props) {
       <div>
       <Section1>
       <NavigationSection1>
-            <Link to='/editrecipe/:id'>
+        <Link to={`/editrecipe/${recipeID}`}>
```

Comment:
Why does there need to be a link to edit within edit steps? (Below too.)

Reply:
agreed this doesn't make much sense

Reply:
You can inject params directly when making a request no need for it to be in history

Reply:
thats inside the back arrow. it redirects to the main edit page

Reply:
after further investigation, I understand why you're passing in the params and I'd say it's an acceptable strategy to tackle the problem but try to find a way without doing it

Reply:
yeah!! I understand. The forms can't get history from props as its only the container component `EditRecipe`  which is wrapped in a <Route/> the steps component aren't. so it's we use <Link/> or useParams/useHistory